src/app.js

```diff
@@ -0,0 +1,17 @@
+const csv = require('csvtojson');
+const fs = require('fs');
+
+import { handleError } from './error-handle.js';
+
+const csvFilePath = './src/csv/node_mentoring_t1_2_input_example.csv';
```

Comment:
Consider using a `process.cwd()` or `__dirname` to provide proper paths whereever
https://stackoverflow.com/questions/9874382/whats-the-difference-between-process-cwd-vs-dirname